frontend/www/js/omegaup/components/problem/creator/cases/CaseEdit.vue

```diff
@@ -689,6 +752,18 @@ export default class CaseEdit extends Vue {
     const oldGroupID: GroupID = this.getSelectedGroup.groupID;
     this.updateCase([oldGroupID, updateCaseRequest]);
   }
+
+  downloadInputFile(ext: '.txt' | '.in') {
+    const caseGroupID: CaseGroupID = {
+      groupID: this.getSelectedGroup.groupID,
+      caseID: this.getSelectedCase.caseID,
+    };
+    const input = this.getStringifiedLinesFromCaseGroupID(caseGroupID);
+    this.$emit('download-input-file', {
+      fileName: this.getSelectedCase.name + ext,
```

Comment:
We prefer to use [`string interpolation`](https://www.educba.com/typescript-string-interpolation/#Syntax):
```suggestion
      fileName: `${this.getSelectedCase.name}${ext}`,
```